### PR TITLE
[DS-2947] DIM crosswalks repeats authority & confidence values in the metadata values

### PR DESCRIPTION
**Pull request by toniprieto**
https://jira.duraspace.org/browse/DS-2947

DIM crosswalk updated to get the authority and confidence values from the two following siblings

### CODE REVIEW

**Review by tdonohue — APPROVED** [review]
👍 Changes look reasonable to me. I'd recommend someone give this a quick test before merging.

**Review by tomdesair — APPROVED** [review]
I was able to reproduce the issue.
I've tested that this fix resolves the problem.

**Comment by tdonohue** [comment]
This has approval & testing. Merging & cherry-picking over to 6.x maintenance branch.

**Pull request merged**
